### PR TITLE
waveshare_epaper: Add 2.90in-dke

### PR DESCRIPTION
**Pull request by polyfloyd**
Pending https://github.com/esphome/esphome/pull/6492

## Description:

**Pull request in [esphome](https://github.com/esphome/esphome) with YAML changes (if applicable):** esphome/esphome#6492

## Checklist:

  - [x] I am merging into `next` because this is new documentation that has a matching pull-request in [esphome](https://github.com/esphome/esphome) as linked above.
    or
  - [ ] I am merging into `current` because this is a fix, change and/or adjustment in the current documentation and is not for a new component or feature.

  - ~~[ ] Link added in `/index.rst` when creating new documents for new components or cookbook.~~

### CODE REVIEW

File: components/status_led.rst [hunk 1]
```diff
@@ -9,7 +9,8 @@ The ``status_led`` hooks into all ESPHome components and can indicate the status
 the device. Specifically, it will:
 
 - Blink slowly (about every second) when a **warning** is active. Warnings are active when for
-  example reading a sensor value fails temporarily or the WiFi/MQTT connections are disrupted.
+  example reading a sensor value fails temporarily, the WiFi/MQTT connections are disrupted, or 
+  if the native API component is included but no client is connected.
```
Comment (clydebarrow) [inline components/status_led.rst]: Shouldn't be changing unrelated files.

Comment (polyfloyd) [reply components/status_led.rst]: Huh, how did that happen?

Comment (polyfloyd) [reply components/status_led.rst]: Fixed

Comment (clydebarrow) [reply components/status_led.rst]: > Huh, how did that happen?

Probably auto-formatting in your editor. Good practice before pushing a PR:

```
git pull upstream dev   (or next as appropriate)
git diff upstream/dev --name-only
```
And check that no files are listed as changed other than the intended ones.

Then after creating the PR do your own review in Github, before anyone else does.

**Review by clydebarrow — APPROVED** [review]
